tests/installation/upgrade/remove_previously_used_repos.pm

```diff
@@ -0,0 +1,19 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles page for detection of previously used repositories.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+
+sub run {
+    my $previous_repo = $testapi::distri->get_previously_used_repos();
+    $previous_repo->get_previously_used_repos();
```

Comment:
test should not contain this getter, this is just a internal, we should show in the test only the actions and validations.  Actually it should be called `_get_previously_used_repos` (in the future we can use a generic name for all the pages, an interface method, is_load or something like that, but not at this point).

Easy to fix if you just call `get_select_for_update` in `press_next` as we used to do for each method in Controllers.